src/qt/CMakeLists.txt

```diff
@@ -250,6 +243,16 @@ if(qt_lib_type STREQUAL "STATIC_LIBRARY")
   )
 endif()
 
+# TODO: The file(GLOB ...) command should be replaced with an explicit
```

Comment:
If we're touching it already, can we fix the TODO, rather than move it? Or, put it into an issue, so that someone might pick it up?

Reply:
> Or, put it into an issue, so that someone might pick it up?

https://github.com/bitcoin/bitcoin/issues/32653.

Reply:
Delete it from here now?

Reply:
Sure! The comment has been deleted.